fasttext/debian/rules

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/make -f
+#export DH_VERBOSE = 1
+
+%:
+	dh $@ --buildsystem=cmake
+
+override_dh_auto_configure:
+	dh_auto_configure -- -DCMAKE_INSTALL_LIBDIR=lib/$(DEB_HOST_MULTIARCH)
```

Comment:
Is this needed?
`dh_auto_configure` doesn't add `-DCMAKE_INSTALL_LIBDIR`?

Reply:
Thanks, It's no need to specify explicitly. removed.